test/e2e/apimachinery/aggregator.go

```diff
@@ -532,7 +531,7 @@ func TestSampleAPIServer(f *framework.Framework, aggrclient *aggregatorclient.Cl
 	if statusCode != 201 {
 		framework.Failf("Flunders client creation response was status %d, not 201", statusCode)
 	}
-	u := &unstructured.Unstructured{}
+	u := &unstructuredv1.Unstructured{}
```

Comment:
unlike the "corev1" this is better as "unstructured" - since it is not v1 of unstructured, but unstructured of metav1